connectors/open_policy_agent/go/api_default.go

```diff
@@ -13,18 +13,54 @@
 package openapiserver
 
 import (
-	"io/ioutil"
 	"log"
 	"net/http"
 
+	opabl "fybrik.io/fybrik/connectors/open_policy_agent/lib"
+	openapiclientmodels "fybrik.io/fybrik/pkg/taxonomy/model/base"
+	"github.com/gdexlab/go-render/render"
 	"github.com/gin-gonic/gin"
 )
 
-// GetPoliciesDecisions - getPoliciesDecisions
-func GetPoliciesDecisions(c *gin.Context) {
+type ConnectorController struct {
+	Policytobeevaluated string
+	Opaserverurl        string
+	Opaclient           *http.Client
+	CatalogReader       *opabl.CatalogReader
+}
+
+func (e *ConnectorController) contactOPA(input openapiclientmodels.PolicyManagerRequest, creds string) (openapiclientmodels.PolicyManagerResponse, error) {
+	opaReader := opabl.NewOpaReader(e.Opaserverurl)
+
+	eval, err := opaReader.GetOPADecisions(&input, creds, e.CatalogReader, e.Policytobeevaluated, e.Opaclient)
+	if err != nil {
+		log.Println("GetOPADecisions err:", err)
```

Comment:
Don't log returned errors. You may use errors.Wrap instead if you need to add information to the error.

Reply:
fixed. Please check. Thanks.